.rubocop.yml

```diff
@@ -294,3 +294,9 @@ RSpec/EmptyLineAfterSubject:
 
 RSpec/ExcessiveDocstringSpacing:
   Enabled: true
+
+RSpec/BeEq:
+  Enabled: true
+
+RSpec/BeNil:
+  Enabled: true
```

Comment:
How about arranging cop names in ASCII order in the department?

```diff
 # RuboCop RSpec cops
+RSpec/BeEq:
+  Enabled: true
+
+RSpec/BeNil:
+  Enabled: true
+
 RSpec/EmptyLineAfterExampleGroup:
   Enabled: true
```

Reply:
Thanks for the review. Updated.